mix.exs

```diff
@@ -68,14 +68,14 @@ defmodule Surgex.Mixfile do
 
   defp optional_deps do
     [
-      {:ecto, "~> 2.1.4"},
+      {:ecto_sql, "~> 3.0"},
       {:ex_marshal, "~> 0.0.8"},
-      {:ex_phone_number, "~> 0.1.1"},
+      {:ex_phone_number, github: "surgeventures/ex_phone_number_new", tag: "v1.0", override: true},
```

Comment:
Same here. `Surgex.PhoneNumber` is deprecated in favor of ecto-phone-number package. Let's not maintain useless crap. And while you're at it, same goes for `Surgex.Config` and `Surgex.Scout` - please remove them all.